### PR TITLE
Add cuda-python

### PR DESCRIPTION
**Pull request by shwina**
Currently required by https://github.com/rapidsai/rmm/pull/930.

cc: @jakirkham for awareness :)

### CODE REVIEW

**Review by ajschmidt8 — APPROVED** [review]

File: conda/recipes/rapids-build-env/meta.yaml [hunk 1]
```diff
@@ -55,6 +55,7 @@ requirements:
     - conda {{ conda_version }}
     - conda-build {{ conda_build_version }}
     - conda-verify {{ conda_verify_version }}
+    - cuda-python ={{ cuda_python_version }}.*
```
Comment (ajschmidt8) [inline conda/recipes/rapids-build-env/meta.yaml:58]:
```suggestion
    - cuda-python {{ cuda_python_version }}.*
```

File: conda/recipes/rapids/meta.yaml [hunk 2]
```diff
@@ -35,6 +35,7 @@ requirements:
   host:
     - python
   run:
+    - cuda-python ={{ cuda_python_version }}.*
```
Comment (ajschmidt8) [inline conda/recipes/rapids/meta.yaml:38]:
```suggestion
    - cuda-python {{ cuda_python_version }}.*
```

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]
From the CI failures, it looks like the extra `=` is causing an `Incorrect version specifier` error.

**Comment by ajschmidt8** [comment]
This PR looks fine, but can we get the dependent PR (https://github.com/rapidsai/rmm/pull/930) passing before we merge this?

**Comment by shwina** [comment]
Apologies for the confusion: that PR depends on this one; not vice-versa :)

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**